intellij-plugin/src/main/resources/messages/ApolloBundle.properties

```diff
@@ -158,6 +158,15 @@ inspection.suppress.field=Suppress for field
 intention.InputConstructorChangeToBuilder.name.editor=Change to builder construction
 intention.InputConstructorChangeToBuilder.name.settings=Change input class constructor to builder
 
+inspection.missingGraphQLDefinitionImport.displayName=Missing GraphQL import
+inspection.missingGraphQLDefinitionImport.reportText=The {0} is not imported
```

Comment:
```suggestion
inspection.missingGraphQLDefinitionImport.reportText=Unresolved {0}: {1}
```
Where `{1}` is the name of the unresolved type/directive

For parity with Kotlin inspections:

![Screenshot 2023-12-20 at 11 26 22](https://github.com/apollographql/apollo-kotlin/assets/3974977/b7939a85-5460-4870-824b-e84d7052e7c9)